apps/fiona/fiona.cpp

```diff
@@ -81,9 +81,15 @@
     // Thus, we do not enable it in this case.
         #define _GLIBCXX_PARALLEL
     #endif  // #if !defined(__MINGW32__) || defined(__amd64__) || defined(__x86_64__) || defined(__ia64__)
+
+    #if defined(__INTEL_COMPILER)
+        #warning "The Intel Compiler crashes with _GLIBCXX_PARALLEL defined (at least until v16.0.2). Therefore, the feature will be disabled."
+        #if __INTEL_COMPILER_BUILD_DATE <= 20160204
```

Comment:
I looked at `icpc -dM -E - < /dev/null` and that was the only reasonable way to determine the version.

Maybe you find a better macro :+1: